libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -628,19 +620,19 @@ void AP_MotorsHeli_Dual::move_actuators(float roll_out, float pitch_out, float c
     float swash2_coll = get_swashplate(2, AP_MOTORS_HELI_DUAL_SWASH_AXIS_COLL, pitch_out, roll_out, yaw_out, collective2_out_scaled);
  
     // get servo positions from swashplate library
-    _servo_out[CH_1] = _swashplate1.get_servo_out(CH_1,swash1_pitch,swash1_roll,swash1_coll);
-    _servo_out[CH_2] = _swashplate1.get_servo_out(CH_2,swash1_pitch,swash1_roll,swash1_coll);
-    _servo_out[CH_3] = _swashplate1.get_servo_out(CH_3,swash1_pitch,swash1_roll,swash1_coll);
-    if (_swashplate1.get_swash_type() == SWASHPLATE_TYPE_H4_90 || _swashplate1.get_swash_type() == SWASHPLATE_TYPE_H4_45) {
-        _servo_out[CH_7] = _swashplate1.get_servo_out(CH_4,swash1_pitch,swash1_roll,swash1_coll);
+    _servo_out[CH_1] = _swashplate[0].get_servo_out(CH_1,swash1_pitch,swash1_roll,swash1_coll);
```

Comment:
To keep this PR simpiler, I have left this basically as it was and manually accessing the swashplate instances.  However, in a future PR i would like to look at pushing this stuff into swashplate.cpp.  We do not need to do the math in swashplate, get the value and send the servo values from this level.  We can make it cleaner.